### PR TITLE
Responsive FAQ Page

### PR DESCRIPTION
**Pull request by Funmi7**
##  What does this PR do
Makes the Faq pages responsive

##  Description of the task to be completed
Add media queries to different screen widths for the faq page

##  How should this manually be tested
Npm start the project and go to the faq page to check it's responsiveness

##  What are the relevant Trello board stories
Responsive FAQ Page

##  Questions (If any):

### CODE REVIEW

**Comment by Funmi7** [comment]
[Feedback page responsive design](https://trello.com/c/ctczha7X/157-feedback-page-responsive-design)

File: src/components/Cards/CoachCard.js [hunk 1]
```diff
@@ -21,6 +22,9 @@ const StyledCoachCard = styled.div`
   background: white;
   box-shadow: 0 6px 10px #d3d3d3;
 
+  /* @media ${devices.mobile} {
+    width: 16rem;
+  } */
```
Comment (BenjaminGrabow) [inline src/components/Cards/CoachCard.js]: Let's delete all comments :)

File: src/views/Marketplace/Marketplace.js [hunk 2]
```diff
@@ -40,6 +46,10 @@ const StyledMarketplace = styled.div`
     width: 100%;
     justify-content: space-evenly;
     padding-top: 1rem;
+
+    /* @media ${devices.mobile} {
+      width: 80%;
+    } */
   }
```
Comment (BenjaminGrabow) [inline src/views/Marketplace/Marketplace.js]: Let's delete also this one, I will merge this after you made the requested changes. Great work Funmi, our app is now almost fully responsive!!!

**Comment by Funmi7** [comment]
[Responsive FAQ Page](https://trello.com/c/xC2BsM45/162-responsive-faq-page)

**Pull request merged**